crates/ide_completion/src/tests/item_list.rs

```diff
@@ -0,0 +1,215 @@
+use expect_test::expect;
+
+use crate::tests::check;
+
+#[test]
+fn in_mod_item_list() {
+    check(
```

Comment:
Let's move the premable appended by `check` into this file, so it's easier to understand where `foo` comes from. Maybe just by defining the `check` function here?